test-e2e/utils/randomizers.ts

```diff
@@ -0,0 +1,9 @@
+const random = (min = 0, max = 0) => min + Math.round(Math.random() * (max - min));
+
+const randomDay = () => random(1, 28);
+
+const randomMonth = () => random(1, 12);
+
+const randomYear = () => random(2050, 2100);
+
+export const randomDate = () => [randomDay(), randomMonth(), randomYear()];
```

Comment:
Won't this sometimes result in invalid dates (i.e. February 30?)

Reply:
The `DateField` component does allow that. But this line of code can't produce that kind of date because `const randomDay = () => random(1, 28);`.